.github/workflows/integration.yml

```diff
@@ -71,7 +71,10 @@ jobs:
 
     - name: Run code coverage
       if: ${{ success() && matrix.os == 'ubuntu-20.04' }}
-      run: bash <(curl -s https://codecov.io/bash)
+      run: |
+        curl -Os https://uploader.codecov.io/latest/linux/codecov
+        chmod +x codecov
+        ./codecov
```

Comment:
Just to check, is a codecov token not needed here?

Reply:
Is it possible to switch to https://github.com/marketplace/actions/codecov instead?

Reply:
From what I know, codecov tokens are [not needed for public repositories](https://docs.codecov.com/docs/frequently-asked-questions#where-is-the-repository-upload-token-found).

I think the reason why we never use `codecov-action` is to avoid comments like [this one](https://github.com/se-edu/addressbook-level3/pull/128#issuecomment-1080217405) appearing everytime we push new commits and run the action.

Reply:
> I think the reason why we never use `codecov-action` is to avoid comments like [this one](https://github.com/se-edu/addressbook-level3/pull/128#issuecomment-1080217405) appearing everytime we push new commits and run the action. Hence, I used the direct replacement for the original bash upload line.

I think the comments are currently [disabled in codecov.yml](https://github.com/reposense/RepoSense/blob/28ab050fa80e9e130ca339e20a452688658f577b/codecov.yml) and not because we are using the bash uploader, or will this action not consider the existence of this file?

Reply:
I tried codecov-action just now. It seems to work - no comments + the `codecov/patch` and `codecov/project` checks are still run and shown.